File: src/3330.find-the-original-typed-string-i.3617/solution.ts
function possibleStringCount (word: string): number {
    let result = 1;
    let index = 0;
    while (index < word.length) {
        const char = word[index++];
        let cnt = 0;
        while (index < word.length && word[index] === char) {
            cnt++;
            index++;
        }
        result += cnt;
    }
    return result;
}

File: src/3330.find-the-original-typed-string-i.3617/solution.java
class Solution {
    public int possibleStringCount(String word) {
        int result = 1;
        char[] chars = word.toCharArray();
        int index = 0;
        while (index<chars.length){
            char c = chars[index++];
            int cnt = 0;
            while (index<chars.length && chars[index] == c){
                cnt++;
                index++;
            }
            result += cnt;
        }
        
        return result;
    }
}